test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -41,6 +41,13 @@ def test_with_content
     host_index_and_show(host)
   end
 
+  def test_no_cves
```

Comment:
```suggestion
  def test_no_content_view_environments
```

CVEs is kinda ambiguous in our context ;-)